src/metabase/api/public.clj

```diff
@@ -75,18 +75,11 @@
   {parameters (s/maybe su/JSONString)}
   (run-query-for-card-with-public-uuid uuid parameters))
 
-(api/defendpoint GET "/card/:uuid/query/json"
-  "Fetch a publically-accessible Card and return query results as JSON. Does not require auth credentials. Public sharing must be enabled."
-  [uuid parameters]
+(api/defendpoint GET "/card/:uuid/query/:export-format-name"
```

Comment:
same thing, probably just shorten this to `export-format` and add a schema validation here